### ISSUE TITLE
Source code cat 2

### ISSUE THREAD

**Issue by Aquillas254** [Aquillas254/alx-system_engineering-devops#1]
import pandas as pd
import numpy as np
df = pd.read_csv('C:/Users/user/Documents/Y2S2/UNISTUDENTS/UNISTUDGRAD.csv')
df
from sklearn.model_selection import train_test_split
from sklearn.tree import DecisionTreeClassifier
from sklearn.metrics import accuracy_score, classification_report, confusion_matrix

#Split the data into feature (x) and labels (y)
x = df.drop('Target', axis=1)
y = df['Target']

#split the data into training and testing sets
x_train, x_test, y_train, y_test = train_test_split(x,y, test_size = 0.2, random_state = 34)
# initialize the decision tree classifier
education = DecisionTreeClassifier(criterion= 'gini', random_state = 10, max_depth=4)

#Train the model on the training data
education.fit(x_train, y_train)
# make predictions on the testing set
y_pred = education.predict(x_test)
y_pred
from sklearn.metrics import accuracy_score, classification_report, confusion_matrix
# Calculate the accuracy of the decision tree classifier
accuracy = accuracy_score(y_test, y_pred)
print('ACCURACY IS: ' , accuracy)

#Evaluate the model's performance
report = classification_report(y_test, y_pred)
print('Classification report:')
print(report)

conf_matrix = confusion_matrix(y_test, y_pred)
print('Confusion Matrix:')
print(conf_matrix)
from sklearn.tree import plot_tree
import matplotlib.pyplot as plt
plt.figure(figsize=(45,40))
plot_tree(education, filled=True,fontsize=14,feature_names=x.columns, class_names=['Dropout','Enrolled','Graduating'])
plt.show()
